docs/_scripts/add_remove.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# Add created RPM content to repository
+echo "Add created RPM Package to repository."
+export TASK_URL=$(http POST $BASE_ADDR$REPO_HREF'modify/' \
+    add_content_units:="[\"$PACKAGE_HREF\"]" | jq -r '.task')
+
+# Poll the task (here we use a function defined in docs/_scripts/base.sh)
+wait_until_task_finished $BASE_ADDR$TASK_URL
+
+# After the task is complete, it gives us a new repository version
+echo "Set REPOVERSION_HREF from finished task."
+export REPOVERSION_HREF1=$(http $BASE_ADDR$TASK_URL| jq -r '.created_resources | first')
```

Comment:
Could it be something more descriptive than  REPOVERSION_HREF1? REPOVERSION_FOX_HREF? (is it the one with the fox package?)  Or REPOVERSION_WITH_PKG_HREF?

Just later when it's used as a base_version, it's not very clear what REPOVESRSION_HREF**1** is.

Reply:
REPOVERSION_HREF_WITH_PKG sounds better to differentiate